Source/WebCore/page/ios/ContentChangeObserver.cpp

```diff
@@ -62,6 +62,11 @@ static bool isHiddenBehindFullscreenElement(const Node& descendantCandidate)
 }
 #endif
 
+static bool isContentChangeObserverEnabled(const Document& document)
```

Comment:
If you make this a method on the class instead of static you can drop the argument entirely and just use m_document, right? I don't see any callers that use it from a static context or with a different document.